packages/hs-react-ui/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -72,12 +73,16 @@ const teaOptions = [
   },
 ];
 
+const StyledPlaceHolder = styled.span`
+  color: blue;
+`;
```

Comment:
while this does demonstrate the capability, I don't think this looks particularly good and in general I think we should be keeping the storybook closer to the defaults, and add in better documentation elsewhere to describe what can be customized